src/main/java/org/fundacionjala/convertor/model/MediaFileModel.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.convertor.model;
+
+import java.io.File;
+import java.util.ArrayList;
+
+
+/**
+ * MediaFileModel.java Copyright (c) 2018 Jalasoft.
+ * 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ * This software is the confidential and proprietary information of
+ * Jalasoft, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+public class MediaFileModel {
```

Comment:
Please put the class documentation
don't forget in the structure put the @author and @version